front/src/common/DotsLoader/index.ts

```diff
@@ -0,0 +1,3 @@
+import DotsLoader from './DotsLoader';
+
+export default DotsLoader;
```

Comment:
```suggestion
export { default as DotsLodaer } form './DotsLoader';
```
same idea for all your index.ts